automation/run-integration-tests.sh

```diff
@@ -33,6 +33,8 @@ docker network create $NET0 || true
 docker network create $NET1 || true
 docker network connect $NET0 $CONTAINER_ID
 docker network connect $NET1 $CONTAINER_ID
+docker exec $USE_TTY -i $CONTAINER_ID /bin/bash -c 'nmcli dev; ip addr'
+docker exec $USE_TTY -i $CONTAINER_ID /bin/bash -c 'ip addr flush eth1 && ip addr flush eth2; ping -c 3 8.8.8.8'
```

Comment:
IMHO `ping -c 1` would be enough but pinging the Google DNS also feels wrong.
Why do you use `&&` to chain the two flushes but ignore the second returncode before pinging Google?

Reply:
The NM debug info should include `nmcli con`.

Reply:
I split the debug from all the rest.
pinging only 1 packet is enough to check if there is a connectivity or not, but I wanted to give it a chance to recover, trying to retry.
Added the `nmcli con`, thanks.